packages/components/organisms/f-offers/stories/mocks/traits/promotionCard2Trait.js

```diff
@@ -0,0 +1,17 @@
+import { trait } from 'miragejs';
+import faker from 'faker';
+
+const type = 'Promotion_Card_2';
+
+/* eslint-disable camelcase */
+export default trait({
+    tt: () => `KFC I Love You Bacon Burger ${faker.datatype.number({ min: 1, max: 10 })}`,
+    ds: () => 'Available Tuesdays',
+    e: () => ({
+        custom_card_type: type,
+        button_1: 'Order now', // faker.random.word(),
+        line_3: 'Order the KFC I Love You Bacon Burger, packed with smoked bacon, baconnaise and sweet Bacon Lovers’ Relish.', // faker.lorem.sentence(),
```

Comment:
Is it worth removing the comments on these lines, or are we planning to revert to faked data at some point? Given this faked data will be available on our public storybook, I'm a bit uncomfortable with mentioning KFC by name there, though we do also have assets making reference to other large chains in the repo already anyhow.

Reply:
I will tidy this up when I put the content cards component in so good call, plan on putting back in fake data